jinahub/indexers/dbms/PostgreSQLIndexer/__init__.py

```diff
@@ -58,22 +69,12 @@ def size(self):
         .. # noqa: DAR201
         """
         with self.handler as postgres_handler:
-            postgres_handler.cursor.execute(f"SELECT COUNT(*) from {self.handler.table}")
+            postgres_handler.cursor.execute(
+                f"SELECT COUNT(*) from {self.handler.table}"
+            )
             records = postgres_handler.cursor.fetchall()
             return records[0][0]
 
-    def post_init(self):
-        """Initialize the PostgresHandler inside the Indexer."""
-        from .postgreshandler import PostgreSQLDBMSHandler
-        super().post_init()
-        self.handler = PostgreSQLDBMSHandler(
-            hostname=self.hostname,
-            port=self.port,
-            username=self.username,
-            password=self.password,
-            database=self.database,
-            table=self.table)
-
     def get_handler(self) -> 'PostgreSQLDBMSHandler':
```

Comment:
These `get_XXX_handler` functions are a smell to me. Just use `self.handler`.

Reply:
I think those were necessary because of the BaseIndexer's close call, which would call these. Removed